roles/provision-metrics-apb/tasks/provision-app-metrics.yml

```diff
@@ -0,0 +1,115 @@
+- name: Postgresql PVC
+  k8s_v1_persistent_volume_claim:
+    name: postgresql-data
+    namespace: "{{ namespace }}"
+    labels:
+      app: app-metrics-api
+      service: prometheus
```

Comment:
should the service label be prometheus?
will that mean the prometheus service will stand load balancing to this too?

Reply:
Yeah I wasn't sure about that, I tagged you in another comment on this PR asking should it be changed, so i'll change that